great_expectations/cli/cli_messages.py

```diff
@@ -10,23 +10,20 @@
              ~ Always know what to expect from your data ~
 </cyan>"""
 
-LETS_BEGIN_PROMPT = """In a few minutes you will see Great Expectations in action on your data!
+LETS_BEGIN_PROMPT = """Let's configure a new Data Context.
 
 First, Great Expectations will create a new directory:
 
     great_expectations
-    |-- expectations
-    |-- great_expectations.yml
-    |-- checkpoints
-    |-- notebooks
-    |   |-- pandas
-    |   |-- spark
-    |   |-- sql
-    |-- plugins
-    |   |-- ...
-    |-- uncommitted
-        |-- config_variables.yml
-        |-- ...
+    ├── great_expectations.yml
+    ├── expectations
+    ├── notebooks
+    ├── plugins
+    ├── .gitignore
+    └── uncommitted
+        ├── config_variables.yml
+        ├── documentation
+        └── validations
```

Comment:
These unicode characters will break windows. Sam changed these to pipes ages ago.

Reply:
@Aylr Thank you - good catch! Fixed.

Reply:
We now need to add a `checkpoints` directory too, correct?

Reply:
It is created in develop and this is in the 011 branch now as well (a merge issue).